tests/Backoffice/Sales/Clients/Application/Create/ClientCreatorTest.php

```
@@ -0,0 +1,62 @@
+
+        $this->shouldPublishDomainEvent($event);
+
+        /** @var ClientRepository&MockInterface $clientRepository */
+        /** @var EventBus&MockInterface $eventBus */
+        $creator = new ClientCreator(
+            $clientRepository,
+            $eventBus
+        );
+
+        ($creator)(new ClientCreatorRequest(
+            $client->id(),
```

PHPStan (2.0.3)

Parameter #1 $id of class MedineTech\Backoffice\Sales\Clients\Application\Create\ClientCreatorRequest constructor expects string, string|null given.


(argument.type)